### DISCUSSION TITLE
Windows UI Library in WPF limitations

### DISCUSSION THREAD

**Discussion by rldribeiro** [microsoft/WinUI-Gallery#878] [Q&A]
Hi! 

Is it possible to run the commands from Windows UI Library in a WPF application targeting .NET Framework 4.8? We are currently working in migrating to .NET 5.

Thanks!

**Comment by marcelwgn** [comment]
I don't think this is possible and if it is possible, it will probably have some bugs. But I haven't tested this myself though.
